examples/experimental/dagster-dlift/dagster_dlift/client.py

```diff
@@ -134,3 +152,69 @@ def get_dbt_tests(self, environment_id: int) -> Sequence[Mapping[str, Any]]:
                 GET_DBT_TESTS_QUERY, {"environmentId": environment_id}, key="tests"
             )
         ]
+
+    def create_dagster_job(self, project_id: int, environment_id: int) -> int:
+        """Creats a dbt cloud job spec'ed to do what dagster expects."""
+        session = self.get_session()
+        response = self.ensure_valid_response(
+            session.post(
+                f"{self.get_api_v2_url()}/jobs/",
+                json={
+                    "account_id": self.account_id,
+                    "environment_id": environment_id,
+                    "project_id": project_id,
+                    "name": get_job_name(environment_id=environment_id, project_id=project_id),
+                    "description": "A job that runs dbt models, sources, and tests.",
+                    "job_type": "other",
+                },
+            ),
+            expected_code=201,
+        ).json()
+        return response["data"]["id"]
+
+    def destroy_dagster_job(self, project_id: int, environment_id: int, job_id: int) -> None:
+        """Destroys a dagster job."""
+        session = self.get_session()
+        self.ensure_valid_response(session.delete(f"{self.get_api_v2_url()}/jobs/{job_id}"))
+
+    def get_job_info_by_id(self, job_id: int) -> Mapping[str, Any]:
+        session = self.get_session()
+        return self.ensure_valid_response(
+            session.get(f"{self.get_api_v2_url()}/jobs/{job_id}")
+        ).json()
+
+    def list_jobs(self, environment_id: int) -> Sequence[Mapping[str, Any]]:
+        return self.make_access_api_request("/jobs/", params={"environment_id": environment_id})[
+            "data"
+        ]
+
+    def trigger_job(self, job_id: int, steps: Optional[Sequence[str]] = None) -> Mapping[str, Any]:
+        session = self.get_session()
+        response = self.ensure_valid_response(
+            session.post(
+                f"{self.get_api_v2_url()}/jobs/{job_id}/run/",
+                json={"steps_override": steps, "cause": "Triggered by dagster."},
+            )
+        )
+        return response.json()
+
+    def get_job_run_info(self, job_run_id: int) -> Mapping[str, Any]:
+        session = self.get_session()
+        return self.ensure_valid_response(
+            session.get(f"{self.get_api_v2_url()}/runs/{job_run_id}")
+        ).json()
+
+    def poll_for_run_completion(self, job_run_id: int, timeout: int = 60) -> int:
+        start_time = time.time()
+        while time.time() - start_time < timeout:
+            run_info = self.get_job_run_info(job_run_id)
+            if run_info["data"]["status"] in {10, 20, 30}:
```

Comment:
nit: would be good to pull these statuses out into constants, or at least leave a comment